vocabularies/scripts/update-csv.py

```diff
@@ -0,0 +1,76 @@
+# This script updates the keys in a given file with the provided values.
+#
+# Depends on Python 3
+#
+# Install dependencies:
+#   pip install -r requirements.txt
+#
+#
+#   Invocation:
+#   cd vocabularies/scripts
+#   python update-csv.py locations spaghetti.csv
+#
+#   Params:
+#    - [which]: Name of csv file to update
+#               (e.g. 'recapCustomerCodes', 'organizations').
+#    - [updateInfo] Name of csv file with values to update
+import sys
+import csv
+
+
+def csv_to_dict(file_name: str) -> dict:
+    information_dict = {}
+    tmp_dict = {}
+
+    try:
+        # Open the CSV file
+        with open(file_name, 'r') as f:
+            raw = f.readlines()
+
+        header = raw[:1][0].strip().split(',')
+        for _, line in enumerate(raw[1:]):
+            tmp = line.replace("\n", "").split(",")
+            current_key = tmp[0]
+            try:
+                for x, key in enumerate(header):
+                    tmp_dict[key] = tmp[x]
+            except Exception as e:
+                if e is None:
+                    print(e)
```

Comment:
hmm! im not quite sure when e would be None. We may just want to print the exception. Something like: 

print(f'Failed to convert csv to dict due to: {e}')

Reply:
Ah yes that was just something I was using in development.

Reply:
wow I did not realize that the except block was actually swallowing `IndexErrors` this whole time from rows missing final commas!